File: grpc-java-examples/src/main/java/com/example/grpc/Delays.java
package com.example.grpc;

import java.util.concurrent.TimeUnit;

public interface Delays {

    static void sleep(int seconds) {
        try {
            TimeUnit.SECONDS.sleep(seconds);
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
            throw new RuntimeException(e);
        }
    }
}

File: grpc-kotlin-examples/src/main/kotlin/com/example/grpc/Delays.kt
package com.example.grpc

import java.util.concurrent.TimeUnit

interface Delays {

  companion object {
    fun sleep(seconds: Int) {
      try {
        TimeUnit.SECONDS.sleep(seconds.toLong())
      } catch (e: InterruptedException) {
        Thread.currentThread().interrupt()
        throw RuntimeException(e)
      }
    }
  }
}
